application/modules/admin/views/scripts/shipment/index.phtml

```diff
@@ -1,25 +1,18 @@
 <link rel="stylesheet" href="<?php echo $this->baseUrl("css/jquery.dataTables.css"); ?>" type="text/css" media="all">
 <style>
 </style>
-<h4 style="width:100%;height:50px;border-bottom:1px solid #777;">
-    <div style="font-size:22.5px;line-height:36px;color:#333;float:left;">Shipments</div>
-</h4>
-<span class="col-md-4 col-xs-12" style="padding:10px 0;">
-    <label>To add a new scheme select a Scheme Type below</label>
-    <select id="scheme" name="scheme" onchange="schemeSelected(this.value);" class="form-control input-sm">
-        <option value=""> -- Select Scheme Type --</option>
-        <?php
-        foreach ($this->schemes as $scheme) { ?>
-        <option value="<?php echo $scheme['scheme_id']; ?>"><?php echo $scheme['scheme_name']; ?></option>
-        <?php
-        } ?>
-    </select>
-</span>
-<span class="col-md-5 col-xs-12" id="btnSpan" style="padding:10px;display:none;">
-    <label>Click on New Shipment to proceed adding Scheme</label>
-    <button class="btn btn-primary btn-sm" onclick="newShipment();" id="enrollBtn"><span>New Scheme</span></button>
-    <button class="btn btn-danger btn-sm" onclick="document.location.href = document.location;"><span>Reset</span></button>
-</span>
+<div class="row" style="width:100%;height:50px;border-bottom:1px solid #777;">
+    <div class="col-md-6">  
+        <h4 >
+            <div style="font-size:22.5px;line-height:36px;color:#333;float:left;">Shipments</div>
+        </h4>
+    </div>
+    <div class="col-md-6 text-right">
+        <input type="hidden" value="<?php echo $this->schemes[0]['scheme_id']; ?>" id="scheme" name="scheme" >
+        <button class="btn btn-primary btn-sm" onclick="newShipment();" ><span>Create New Shipment</span></button>
```

Comment:
Please change the text from "Create New Shipment" to "Create Panel" and add a + icon in the button? I'd like it to look more like the button for adding a new survey with a slightly larger button, plus icon and some margin beneath this row div:
![image](https://user-images.githubusercontent.com/3584031/108331647-a67c3f80-71d7-11eb-9c8b-aa75ed359d9a.png)

Reply:
Looking into the problem right now.